client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -204,14 +206,35 @@
     </b-modal>
 
     <b-modal
-      :title="$t('changeBlock')"
       size="xl"
       :visible="showEditor"
       body-class="p-0 border-top-0"
       footer-class="d-flex justify-content-between"
       header-class="p-3 pb-0 border-bottom-0"
       @hide="editor = undefined"
     >
+      <template #modal-title>
+        <div class="d-flex gap-1 align-items-center">
+          <h5 class="mb-0">
+            {{ $t('changeBlock') }}
+          </h5>
+
+          <font-awesome-icon
```

Comment:
Its currently always shown, there is no v-if. The icon should only be there if its actually a clone